app/controllers/concerns/v2/metadata.rb

```diff
@@ -0,0 +1,122 @@
+# frozen_string_literal: true
+
+module V2
+  # Sets up the metadata to be passed fo fast_jsonapi to be
+  # added to our API responses
+  module Metadata
+    extend ActiveSupport::Concern
+
+    included do
+      before_action :set_headers
+
+      def set_headers
+        response.headers['Content-Type'] = 'application/vnd.api+json'
+      end
+
+      # This is part of a JSON schema, no need for strict metrics
+      # rubocop:disable Metrics/MethodLength
+      # rubocop:disable Metrics/AbcSize
+      def metadata(opts = {})
+        opts[:total] ||= count_collection
+
+        {
+          meta: {
+            total: opts[:total],
+            self.class::SEARCH => permitted_params[self.class::SEARCH],
+            tags: tags,
+            limit: pagination_limit,
+            offset: pagination_offset,
+            sort_by: permitted_params[:sort_by],
+            relationships: relationships_enabled?
```

Comment:
This field can be removed, we won't use it in V2, at least not in its current form